server/plugin.go

```diff
@@ -1,304 +0,0 @@
-package main
-
-import (
-	"encoding/json"
-	"fmt"
-	"net/url"
-	"sort"
-	"strings"
-	"sync"
-
-	"github.com/mattermost/mattermost-server/model"
-	"github.com/mattermost/mattermost-server/plugin"
-)
-
-type Plugin struct {
-	plugin.MattermostPlugin
-
-	// configurationLock synchronizes access to the configuration.
-	configurationLock sync.RWMutex
-
-	// configuration is the active plugin configuration. Consult getConfiguration and
-	// setConfiguration for usage.
-	configuration *configuration
-
-	currentAnalytic *Analytic
-
-	cronSavePoison chan bool
-}
-
-const CommandTrigger = "analytics"
-
-func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	if !strings.HasPrefix(args.Command, "/"+CommandTrigger) {
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         fmt.Sprintf("Unknown command: " + args.Command),
-		}, nil
-	}
-
-	if strings.Contains(args.Command, "new") {
-		p.newSession()
-		allSessions, _ := p.allSessions()
-		j2, _ := json.Marshal(allSessions)
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL,
-			Text:         string(j2),
-		}, nil
-	}
-
-	if strings.Contains(args.Command, "session") {
-		allSessions, _ := p.allSessions()
-		j2, _ := json.Marshal(allSessions)
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL,
-			Text:         string(j2),
-		}, nil
-	}
-
-	if strings.Contains(args.Command, "raw") {
-		j2, _ := json.Marshal(p.currentAnalytic)
-		return &model.CommandResponse{
-			ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL,
-			Text:         string(j2),
-		}, nil
-	}
-
```

Comment:
You should add a `help` command.

Reply:
Now /analytics do only one thing : return analytics. I think an help command is not necessary  https://github.com/manland/mattermost-plugin-analytics/pull/3/files#diff-7e42e9629fece6c3b56011e7de76423bL40